src/utils/table-columns.ts

```diff
@@ -163,7 +163,7 @@ export function setAllColumnWidths(
                         getIconWidth(targetEl) +
                         35
                     // Set the width of the column
-                    targetEl.style.minWidth =
+                    targetEl.style.width =
```

Comment:
yes, we can set both since minWidth has no effect any way. it will not make it any less breaking though.

Reply:
It might affect the existing tables that the width will work, can you verify which tables will be affected in OA (if at all) ?
Audio studio is also using the table

Reply:
sure, I will go over the pages in local and rc, and make screenshot pairs.

Reply:
ok I tried to take these screenshots, and the results are mixed, e g d.m./Datasets tab looks better than before:
<img width="1358" alt="Screenshot 2024-11-20 at 3 19 01 PM" src="https://github.com/user-attachments/assets/77546dba-d0ce-4c4a-94be-c53f33a64203">
<img width="1351" alt="Screenshot 2024-11-20 at 3 19 13 PM" src="https://github.com/user-attachments/assets/056de9c1-d380-4ad1-9d23-4c2f9cb798cf">
while d.m./Storage Drivers looks worse:
<img width="1357" alt="Screenshot 2024-11-20 at 3 19 29 PM" src="https://github.com/user-attachments/assets/58be9e67-fb5e-43ea-b980-471463d7f0d1">
<img width="1340" alt="Screenshot 2024-11-20 at 3 20 06 PM" src="https://github.com/user-attachments/assets/0f32c4a8-961f-4d58-8197-0ffe56f69a2c">
so for now let's shelve this, I will find another way to fix d.m./Embeddings tab alone